src/test/java/com/jwplayer/southpaw/util/ByteArraySetTest.java

```diff
@@ -16,14 +16,40 @@
 package com.jwplayer.southpaw.util;
 
 import org.apache.commons.lang.NotImplementedException;
+import org.apache.commons.lang.RandomStringUtils;
 import org.junit.Test;
 
+import scala.Tuple2;
+
 import java.util.*;
 
 import static org.junit.Assert.*;
 
 
 public class ByteArraySetTest {
+    private static final int CHUNK_MAX_SIZE = 4096;
```

Comment:
This value already appears as a constant [here](https://github.com/jwplayer/southpaw/blob/master/src/main/java/com/jwplayer/southpaw/util/ByteArraySet.java#L77) can we use this rather than hardcode it in two places in case this ever changes?

Reply:
Updated